### PR TITLE
Allow for overwriting all files in shell

### PR DESCRIPTION
**Pull request by dakota**
If you are creating multiple files you should be able to overwrite all of them after the first prompt to overwrite.

### CODE REVIEW

**Comment by markstory** [comment]
Nice, we should have a test for this.

**Comment by markstory** [comment]
Ping any updates on this @dakota ?

**Comment by dakota** [comment]
Oh, kinda forgot about this! Will get on it asap!

**Comment by dakota** [comment]
Tests added. Not sure why appveyor is failing.

**Comment by antograssiot** [comment]
@dakota Appveyor build is broken due to a bug in "phpunit/phpunit-mock-objects": "2.3.2"

File: tests/TestCase/Console/ShellTest.php [hunk 1]
```diff
@@ -526,6 +526,39 @@ public function testCreateFileOverwriteNonInteractive()
...
+            $path . DS . 'file3.php' => 'My third content'
+        ];
+
+        new Folder($path, true);
+
+        $this->io->expects($this->once())
+            ->method('askChoice')
+            ->will($this->returnValue('a'));
+
+        foreach ($files as $file => $content) {
```
Comment (markstory) [inline tests/TestCase/Console/ShellTest.php]: You never use `$content`.

Comment (dakota) [inline tests/TestCase/Console/ShellTest.php]: Oops! Missed that. Thanks, will quickly fix it.

**Comment by dakota** [comment]
Failure looks like one of those random failures

**Comment by markstory** [comment]
Nice work @dakota

**Pull request merged**